securedrop/manage.py

```diff
@@ -148,16 +148,16 @@ def _make_password():
 def _add_user(is_admin=False):
     username = _get_username()
 
-    print("Note: Journalist passwords are now autogenerated.")
+    print("Note: Passwords are now autogenerated.")
     password = _make_password()
-    print("This journalist's password is: {}".format(password))
+    print("This admin's password is: {}".format(password))
```

Comment:
Recall we also have a `manage.py` command for adding journalist accounts:

```py
    journalist_subp_a = subps.add_parser('add_journalist', help='^')
    journalist_subp_a.set_defaults(func=add_journalist)
```

So the messages here should vary on the value of kwarg `is_admin`.